core/src/main/resources/error/error-classes.json

```diff
@@ -219,6 +219,11 @@
           "Input to the function <functionName> cannot contain elements of the \"MAP\" type. In Spark, same maps may have different hashcode, thus hash expressions are prohibited on \"MAP\" elements. To restore previous behavior set \"spark.sql.legacy.allowHashOnMapType\" to \"true\"."
         ]
       },
+      "INTERNAL_ERROR" : {
```

Comment:
I don't think we should introduce such sub-class. Could you re-use the existing `INTERNAL_ERROR`:
https://github.com/apache/spark/blob/31b923d50fa6176312f7217069c0055cd778788f/core/src/main/resources/error/error-classes.json#L557-L561

Reply:
Done